app/src/main/java/com/example/taskintern/view/WONavCompFirsFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.taskintern.view
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.taskintern.R
+import com.example.taskintern.adapter.Adapter
+import com.example.taskintern.databinding.FragmentWONavCompFirsBinding
+import com.example.taskintern.model.Weather
+
+
+class WONavCompFirsFragment : Fragment() {
+    private lateinit var binding: FragmentWONavCompFirsBinding
+    private lateinit var adapter: Adapter
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
Binding'i lateinit tanimladigindan nullable belirtecini kaldirabiliriz
```suggestion
    ): View {
```